frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Edit HPA action
+	As a user, I want to edit the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment 
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user is in topology
+
+@regression
+Scenario: Edit HPA action form view
+   Given user has a workload with HPA assigned to it
```

Comment:
Include the test data wherever applicable i.e, workload as below
user has a workload "nodejs-ex-git-1" with HPA assigned to it